From 481944ba3e85b9a5d1aeaff0303d98c2d0d8f111 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 14 Sep 2021 17:56:41 +0300
Subject: [PATCH] [SW-0] gaudi2/greco: arc: staticize copy_preboot()

The copy_preboot() function is not used outside its file, so make it
static.

Change-Id: Ifc7357f2d3150ff54fb901e1f9fb320f1cb2a0b4
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/arc/common.c b/arch/gaudi2/arc/common.c
index 42d4f73..c729040 100644
--- a/arch/gaudi2/arc/common.c
+++ b/arch/gaudi2/arc/common.c
@@ -225,7 +225,7 @@
 	return ARCBOOT_BRINGUP_OK;
 }
 
-u64 copy_preboot(void)
+static u64 copy_preboot(void)
 {
 	u32 config_fuse;
 	int ret;
diff --git a/arch/greco/arc_elf32/common.c b/arch/greco/arc_elf32/common.c
index 1c1c171..4145d26 100644
--- a/arch/greco/arc_elf32/common.c
+++ b/arch/greco/arc_elf32/common.c
@@ -225,7 +225,7 @@
 	return ARCBOOT_BRINGUP_OK;
 }
 
-u64 copy_preboot(void)
+static u64 copy_preboot(void)
 {
 	u32 config_fuse;
 	int ret;
